pyte/screens.py

```diff
@@ -1043,6 +1048,26 @@ def debug(self, *args, **kwargs):
         By default is a noop.
         """
 
+    def scroll_up(self, lines):
+        """Scroll up `lines` lines."""
+        lines_to_scroll = range(self.margins.top, self.margins.bottom + 1)
```

Comment:
We should handle when margins is None.
```suggestion
        margins = self.margins or Margins(0, self.lines - 1)
        lines_to_scroll = range(margins.top, margins.bottom + 1)
```